sanic/handlers.py

```diff
@@ -47,9 +61,13 @@ def response(self, request, exception):
         :param exception: Exception to handle
         :return: Response object
         """
-        handler = self.handlers.get(type(exception), self.default)
+        handler = self.lookup(exception)
         try:
-            response = handler(request=request, exception=exception)
+            response = handler and handler(
```

Comment:
if handler is `self._missing` which it will be due to line 43, then you will get `TypeError: 'object' object is not callable`. Also what is the purpose of the `and`? I feel that obfuscates whatever intention you have here.